observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc

```diff
@@ -80,3 +90,5 @@ include::modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc[l
 * xref:../../../operators/admin/olm-adding-operators-to-cluster.adoc#olm-installing-from-operatorhub-using-web-console_olm-adding-operators-to-a-cluster[Installing from OperatorHub using the web console]
 * xref:../../../operators/user/olm-creating-apps-from-installed-operators.adoc#olm-creating-apps-from-installed-operators[Creating applications from installed Operators]
 * xref:../../../cli_reference/openshift_cli/getting-started-cli.adoc#getting-started-cli[Getting started with the OpenShift CLI]
+
+
```

Comment:
Remove this extra whitespace

Reply:
Thank you for catching this. I've removed both of these blank lines.